src/pages/ReviewPage/Component/FilterGenreMenu.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import MenuList from './MenuList';
+import API_URLS from '../../../config';
+import './FilterGenreMenu.scss';
+
+export default class FilterGenreMenu extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isMenuChecked: false,
+      genreMenus: {},
+    };
+  }
+
+  componentDidMount() {
+    fetch(API_URLS.REVIEW_GENRE)
+      .then(res => {
+        if (res.status === 200) {
+          return res.json();
+        }
+      })
+      .then(res => this.setState({ genreMenus: res['genre_list'] }));
+  }
+
+  clickRandomMenu = () => {
+    this.props.getMovieData();
+    this.setState({
+      isMenuChecked: !this.state.isMenuChecked,
+    });
+  };
+
+  render() {
+    const { getMovieData, closeModal } = this.props;
+    const { isMenuChecked, genreMenus } = this.state;
+    const genreMenusArr = Object.entries(genreMenus);
+    return (
+      <section className="filterGenreMenu">
+        <header className="menuTitle">
+          <button className="menuClosedBtn" onClick={closeModal}>
+            X
+          </button>
+          영화
+        </header>
+        <ul className="menuLists">
+          <li className="menuList" onClick={this.clickRandomMenu}>
+            랜덤 영화 {isMenuChecked && <span className="menuChecked">✔</span>}
+          </li>
+          <h6 className="menuSubTitle">장르</h6>
+          {genreMenusArr.map(el => (
+            <MenuList
+              getMovieData={getMovieData}
+              key={el[0]}
+              id={el[0]}
+              genre={el[1]}
+            />
+          ))}
```

Comment:
좀 더 명확한 네이밍을 위해서 배열구조분해할당 해주면 좋겠네요
정확히 어떤 데이터인지는 몰라서 유추해서 지어봤습니다 확인하시고 의도하신바에 맞게 명확하게 지어주세요!
```suggestion
          {genreMenusArr.map(([id, genre]) => (
            <MenuList
              getMovieData={getMovieData}
              key={id}
              id={id}
              genre={genre}
            />
          ))}
```

Reply:
ㅋㅋㅋㅋ유추하신게 맞아요 감사합니다! 수정했어요